src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -319,6 +319,10 @@ function RemoteFunctions(experimental) {
             if (this.trigger) {
                 _trigger(element, "highlight", 1);
             }
+            
+            if (!window.event) {
+                element.scrollIntoViewIfNeeded();
```

Comment:
According to the current version of [MDN](https://developer.mozilla.org/it/docs/Web/API/Element/scrollIntoViewIfNeeded), this function is supported only in Chrome, Opera and Safari.
I'm not familiar with LivePreview but this file is injected only in case of LivePreview with Chrome, or it is also used with MultiBrowserLivePreview?
In the seconda case, is it possible to change it to use `scrollIntoView` instead?